index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Capstone Project 1</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+</head>
+<body>
+	<header class="container">
+		<div class="sub-container1">
+			<ul class="link1">
+				<li class="list-item"><a href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="list-item"><a href="#"><i class="fa fa-twitter"></i></a></li>
+				<li class="list-item"><a href="#">English</a></li>
+				<li class="list-item"><a href="#">My Page</a></li>
+				<li class="list-item"><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="sub-container2">
+			<div class="logo">
+				<a href="index.html"><img src="./images/logo2.png" alt=""></a>
+			</div>
+			<ul class="link2" id="nav">
+				<li class="list-item2 mobile"><a href="index.html">Home</a></li>
+				<li class="list-item2"><a href="about.html">About</a></li>
```

Comment:
- [ ] Kindly 😃 try to add the nav links of the header at the nav tag as I see you didn't put them in the nav tag, so please try to use the nav tag to wrap the ul of the links to stick to the semantics and this will be great for the screen readers 🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/226697860-7aae1ddf-3216-4361-8262-6b34e2436d13.png)   |   ![image](https://user-images.githubusercontent.com/83360636/226697796-1a386d9c-9774-449c-a94f-ef345909c66d.png)  |